src/screens/SiteDetailScreen.js

```diff
@@ -430,12 +472,27 @@ class SiteDetailScreen extends Component {
                                 {description}
                             </Text>
 
+                            <Text style={sectionTitleStyle}>
+                                {campsite_form.facilities}
+                            </Text>
+                            <View>
+                                {this.renderFacilities(facilities)}
+                            </View>
+
+                            <Text style={sectionTitleStyle}>
+                                {campsite_form.features}
+                            </Text>
+                            <View>
+                                {this.renderFeatures(features)}
+                            </View>
+
```

Comment:
Here, and below, I changed the order that we show things in the `SiteDetailScreen` to make it flow more in the order that a user would actually care about.